app/src/main/res/layout/cover_fragment.xml

```diff
@@ -10,6 +11,12 @@
     android:padding="8dp"
     android:gravity="center">
 
+    <FrameLayout
```

Comment:
Feels a bit strange to use a FrameLayout here. Will have a closer look in the next few days.

Reply:
we can use Space